test/integration_test.rb

```diff
@@ -160,9 +172,9 @@ def sign_params(params)
     params
   end
 
-  def expect_access_token_request(access_token)
+  def expect_access_token_request(access_token, scope)
     FakeWeb.register_uri(:post, "https://snowdevil.myshopify.com/admin/oauth/access_token",
-                         body: JSON.dump(access_token: access_token),
+                         body: JSON.dump(access_token: access_token, scope: scope),
```

Comment:
We are now returning this in the response?  If so, then we should update the documentation (https://docs.shopify.com/api/authentication/oauth) which currently says:

> The server will respond with an access token.
>
> ```
{
  "access_token": "f85632530bf277ec9ac6f649fc327f17"
}
```